File: espressobucks/template/espressobucks.py
def solve(n: int, m: int, grid: list[list[chr]]) -> list[list[chr]]:
    return None

if __name__ == '__main__':
    n, m = map(int, input().split())
    grid = [list(input()) for _ in range(n)]
    answer = solve(n, m, grid)
    for row in answer:
        print(''.join(row))

File: espressobucks/template/espressobucks.java
import java.util.*;

public class espressobucks {
    private static char[][] solve(int n, int m, char[][] grid) {
        return null;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int m = in.nextInt();
        char[][] grid = new char[n][m];
        for (int i = 0; i < n; i++) {
            String line = in.next();
            for (int j = 0; j < m; j++) {
                grid[i][j] = line.charAt(j);
            }
        }
        in.close();

        char[][] answer = solve(n, m, grid);
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                System.out.print(answer[i][j]);
            }
            System.out.println();
        }
    }
}
